app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -45,6 +47,14 @@
     end
   end
 
+  if ga4_tracking
+    data_attributes[:show_all_attributes] = {
```

Comment:
Does this need a bit more in the JS test to check that these attributes get copied onto the show/hide all button correctly? (I haven't checked, there might already be something from the previous iteration, don't know if it would need updating or not).